tests/Mapper.py

```diff
@@ -45,34 +55,63 @@ def tearDown(self):
 
     def testFileNameInterpretation(self):
         pfsParseTask = PfsParseTask(name='name')
-        site = 'S'
-        category = 'A'
-        visit = 4
-        arms = ['b', 'r', 'n', 'm']
-        minArmNum = 1
-        maxArmNum = 4
-        minSpectrographNum = 1
-        maxSpectrographNum = 4
-        for armNum in np.arange(minArmNum, maxArmNum + 1, 1):
-            for spectrograph in np.arange(minSpectrographNum, maxSpectrographNum + 1, 1):
-                ccd = spectrograph - 1
-                if int(armNum) == minArmNum + 1 or int(armNum) == minArmNum + 3:
-                    ccd += 4
-                elif int(armNum) == minArmNum + 2:
-                    ccd += 8
+        mapper = PfsMapper(
+            root=os.path.join(lsst.utils.getPackageDir('drp_stella_data'),'tests/data/PFS')
+        )
```

Comment:
fixed to run tests only if drp_stella_data is setup, show warning message on screen if not setup